test/saml20_clj/sp_test.clj

```diff
@@ -1,7 +1,9 @@
 (ns saml20-clj.sp-test
   (:require [clojure.test :refer :all]
             [clj-time.core :as ctime]
-            [saml20-clj.sp :refer :all]))
+            [saml20-clj.sp :refer :all]
+            [saml20-clj.xml :refer all])
```

Comment:
`Exception in thread "main" java.lang.Exception: :only/:refer value must be a sequential collection of symbols, compiling:(saml20_clj/sp_test.clj:1:1)`

missing `:` before all.

meanwhile, i still suggest moving this to a separate ns